pertpy/plot/_dialogue.py

```diff
@@ -0,0 +1,75 @@
+import os
+from typing import Literal, Optional, Union
+
+import matplotlib.pyplot as plt
+import pandas as pd
+import scanpy as sc
+import seaborn as sns
+from anndata import AnnData
+
+
+class DialoguePlot:
+    @staticmethod
+    def split_violins(
+        adata: AnnData,
+        split_key: str,
+        celltype_key=str,
+        split_which: (str, str) = None,
+        mcp: str = "mcp_0",
+    ) -> plt.Axes:
+        """
+        Usage: Plots split violin plots for a given MCP and split variable. Any cells
+        with a value for split_key not in split_which are removed from the plot.
+
+        Args:
+            adata (AnnData): Annotated data object.
```

Comment:
Please don't add typehints in any arguments. vscode or something probably autogenerates them, but we don't need them because a sphinx extension gets them from the function definition. Having them twice just adds maintenance burden.